src/abstractdataframe/show.jl

```diff
@@ -173,20 +173,15 @@ function getprintedwidth(maxwidths::Vector{Int}) # -> Int
 end
 
 """
-    getchunkbounds(maxwidths::Vector{Int},
+    getchunkedcols(maxwidths::Vector{Int},
                    splitcols::Bool,
                    availablewidth::Int)
 
-When rendering an `AbstractDataFrame` to a REPL window in chunks, each of
-which will fit within the width of the REPL window, this function will
-return the indices of the columns that should be included in each chunk.
-
-NOTE: The resulting bounds should be interpreted as follows: the
-i-th chunk bound is the index MINUS 1 of the first column in the
-i-th chunk. The (i + 1)-th chunk bound is the EXACT index of the
-last column in the i-th chunk. For example, the bounds [0, 3, 5]
-imply that the first chunk contains columns 1-3 and the second chunk
-contains columns 4-5.
+When rendering an `AbstractDataFrame` to a REPL window in chunks, each of which
+will fit within the width of the REPL window, this function will return an
+array of chunk column indices that should be included in each chunk. For
+example, if printing is divided into two chunks containing columns 1 through 3
+and 4 through 5 respectively, `getchunkedcols` will return [[1, 2, 3], [4, 5]].
```

Comment:
after reviewing the code it seems that it is better to return a vector of `UnitRange` - this is not a big deal as this method does not have to be super efficient, but it seems cleaner to return `[1:3, 4:5]` in your example - for sure it will allocate less (so it would be a better style).